pkg/manifest/os.go

```diff
@@ -822,6 +826,25 @@ func (p *OS) serialize() osbuild.Pipeline {
 		}
 	}
 
+	if len(p.CACerts) > 0 {
+		for _, cc := range p.CACerts {
```

Comment:
I would love to see a test for this but I understand this area is really difficult to test so fine to ignore that (but I also can't help to mention it)